tests/test_fmi3.py

```diff
@@ -0,0 +1,13 @@
+# TODO: Remove/rename this file
+import sys
+from pathlib import Path
+
+this_dir = Path(__file__).parent.absolute()
+sys.path.append(str(this_dir))
+
+from conftest import setup_reference_fmus
```

Comment:
This line can actually be skipped, all fixtures in conftest.py are automatically imported.

Reply:
I removed most stuff above now, see my latest commit